### PR TITLE
amp-carousel: clear autoplay after user interaction

### PR DESCRIPTION
**Pull request by zhouyx**
I tested on an android phone. What other devices I need to test?

### CODE REVIEW

**Comment by zhouyx** [comment]
tested on Andriod chrome, ios safari & chrome

File: extensions/amp-carousel/0.1/slidescroll.js [hunk 1]
```diff
@@ -156,6 +156,9 @@ export class AmpSlideScroll extends BaseSlides {
    */
   touchMoveHandler_() {
     this.clearAutoplay();
+    if (!this.hasNativeSnapPoints_) {
+      return;
+    }
```
Comment (sriramkrish85) [inline extensions/amp-carousel/0.1/slidescroll.js]: we should combine -  cancelTouchEvents_() here .

Comment (zhouyx) [inline extensions/amp-carousel/0.1/slidescroll.js]: I feel `cancelTouchEvent_()` should be called only once. But `touchMoveHandler_` is called everytime we receive `touchmove` event

Comment (sriramkrish85) [inline extensions/amp-carousel/0.1/slidescroll.js]: cancel timeout registers a touch move handler - which does some thing - combine it here.

Comment (sriramkrish85) [inline extensions/amp-carousel/0.1/slidescroll.js]: nvm - they are on different elements

**Pull request merged**